src/Backend/Modules/MediaGalleries/Js/MediaGalleries.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Interaction for the media galleries module
+ * global: jsBackend
+ */
+jsBackend.mediaGalleries =
```

Comment:
can you give the commit hashes as a comment when you have done something? makes it more easy to check the fix